TypeCobol/Compiler/Parser/CodeElementBuilder/CobolCodeElementBuilder.cs

```diff
@@ -2280,5 +2280,10 @@ public override void EnterTruncateStatement([NotNull] CodeElementsParser.Truncat
             Context = context;
             CodeElement = _sqlCodeElementBuilder.CreateTruncateStatement(context);
         }
-    }
+        public override void EnterWhenEverStatement([NotNull] CodeElementsParser.WhenEverStatementContext context)
+        {
+            Context = context;
+            CodeElement = _sqlCodeElementBuilder.CreateWhenEverStatement(context);
+        }
+	}
```

Comment:
indent (but this file uses mixed indent so i know it's complicated to catch all errrors at first sight)